### DISCUSSION TITLE
Livewire v3 : Pagination Bug on lazy component.

### DISCUSSION THREAD

**Discussion by sajjadhossainshohag** [livewire/livewire#5958] [7. Bugs (archived)]
When I am trying to show Livewire component in blade file. Then, pagination with URL query parameters worked fine. But when the Livewire component tries idle mode the query parameter is not visible in the URL.

**My Code Below:**
![code](https://github.com/livewire/livewire/assets/63788037/d5331b4a-a540-437e-a20c-45b007bda225)

**With Lazy:**
![Screenshot 2023-07-25 182227](https://github.com/livewire/livewire/assets/63788037/e9ddc478-573c-4bf9-9d49-0bf3493ad1b6)

**Without Lazy:**
![Screenshot 2023-07-25 182244](https://github.com/livewire/livewire/assets/63788037/401f6ec6-ca33-467e-91fa-ab70bf246c5d)

**Comment by sajjadhossainshohag** [comment]
@joshhanley  Hello dear, please help me with this.

**Reply by joshhanley** [reply]
@sajjadhossainshohagbd while I appreciate that problems are frustrating, please don't just tag people. We're currently flat-out fixing bugs as it's a beta release, so please be patient (and I was asleep when you submitted your bug report and when you tagged me) and we will get to it when we can 🙂

**Comment by joshhanley** [comment]
@sajjadhossainshohagbd thanks for reporting! Yeah that seems like a bug. Any chance you could submit a PR with a failing test so we can investigate? Thanks!
